test/e2e/helpers.js

```diff
@@ -187,7 +185,16 @@ async function withFixtures(options, testSuite) {
   const bundlerServer = new Bundler();
   const https = await mockttp.generateCACertificate();
   const mockServer = mockttp.getLocal({ https, cors: true });
-  let numberOfDapps = dapp ? 1 : 0;
+  const dappOpts = dappOptions || {};
+  const hasCustomPaths =
+    Array.isArray(dappOpts.customDappPaths) &&
+    dappOpts.customDappPaths.length > 0;
+  const customCount = hasCustomPaths ? dappOpts.customDappPaths.length : 0;
+  const defaultCount =
+    typeof dappOpts.defaultTestDapp === 'number' && dappOpts.defaultTestDapp > 0
```

Comment:
I don't find the param name very intuitive. Should the name be `numberOfTestDapps`? Or something specifying more clearly what it does?